core-api/src/main/java/com/optimizely/ab/event/internal/payload/DecisionMetadata.java

```diff
@@ -0,0 +1,93 @@
+/**
+ *
+ *    Copyright 2020, Optimizely and contributors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.optimizely.ab.event.internal.payload;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.optimizely.ab.annotations.VisibleForTesting;
+
+public class DecisionMetadata {
+    @JsonProperty("flag_type")
+    String flagType;
+    @JsonProperty("flag_key")
+    String flagKey;
+    @JsonProperty("variation_key")
+    String variationKey;
+
+    @VisibleForTesting
+    public DecisionMetadata() {
+    }
+
+    public DecisionMetadata(String flagType, String flagKey, String variationKey) {
+        this.flagType = flagType;
+        this.flagKey = flagKey;
+        this.variationKey = variationKey;
+    }
+
+    public String getFlagType() {
```

Comment:
Any reason why code indentation is different than `getFlagKey ` and `getVariationKey ` methods below?